packages/quantic/force-app/examples/main/lwc/exampleQuanticRefineModalContent/exampleQuanticRefineModalContent.html

```diff
@@ -70,7 +70,6 @@
             <c-quantic-timeframe
               amount="10"
               unit="year"
-              label="Last decade"
```

Comment:
looks like labels aren't being replicated properly in the refine modal. But i'm assuming the tests once passed so is it a regression from a recent change?